scripts/spread-hunter-menu.ps1

```
@@ -866,6 +930,23 @@ function Show-Status {
         Write-FileRow -Label "PID file" -Status "MISSING" -Path "runtime/shadow-dash.pids.json" -Dynamic "no PID file"
     }
 
+    # ── 1c · SHADOW RUN (recorded session: screener + rehearsal loop + scoped watcher) ──
+    $sess = $null
+    if (Test-Path $ShadowSessionFile) { try { $sess = Get-Content $ShadowSessionFile -Raw | ConvertFrom-Json } catch {} }
```

PSScriptAnalyzer (1.25.0)

[warning] Empty catch block is used. Please use Write-Error or throw statements in catch blocks.

(PSAvoidUsingEmptyCatchBlock)